activiti-cloud-service-common/activiti-cloud-services-dbp-rest/src/main/resources/config/alfresco-rest-config.properties

```diff
@@ -1,3 +1,6 @@
 spring.data.rest.default-page-size=100
 spring.data.rest.default-media-type=application/json
 spring.hateoas.use-hal-as-default-json-media-type=false
+
+max_items_limit=${MAX_ITEMS_LIMIT:1000}
+max_items_limit_enabled=${MAX_ITEMS_LIMIT_ENABLED:true}
```

Comment:
Is this extra property needed? limit=0 could mean "no limit"

Reply:
we decided to add a limit to the maxItems parameter to 1000 elements, but if there is the need to tweak that i also added an env variable to set a different limit. 0 would be 0 items limit in this case so no items will be retrieved, to disable the maxLimit there is the MAX_ITEMS_LIMIT_ENABLED, if set to false it will disable the check.

Reply:
I understood that, I mean that limit=0 could be handled in a specific way (eg no limit) because there are no use cases where "0 items" configuration would be useful/meaningful anyway.
But that's kind of a detail, I don't see the point of this extra property that's all.